docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md

```diff
@@ -59,6 +61,167 @@ Regardless of the alert level, you create an alert as follows:
+  ```
+
+- ```expression
+  expr: max by (node_prefix) (changes(node_boot_time{node_prefix=""}[30m])) > 0
+  ```
+
+- ```expression
+  expr: max by (node_prefix) (changes(node_boot_time{node_prefix=""}[30m])) > 2
+  ```
+
+- ```expression
+  expr: last_over_time(ybp_alert_manager_status{customer_uuid = ""}[1d]) < 1
+  ```
+
+- ```expression
+  expr: sum by (node_prefix) (increase(majority_sst_files_rejections{node_prefix=""}[10m])) > 0
+  ```
+
+- ```expression
+  expr: max by (node_prefix) (count by (node_prefix, exported_instance) (max_over_time(yb_node_underreplicated_tablet{node_prefix=""}[5m])) > 0)
+  ```
+
+- ```expression
+  expr: max by (node_prefix) (changes(yb_node_boot_time{node_prefix=""}[30m]) and on (node_prefix) (max_over_time(ybp_universe_update_in_progress{node_prefix=""}[31m]) == 0)) > 0
+  ```
+
+- ```
+  expr: max by (node_prefix) (changes(yb_node_boot_time{node_prefix=""}[30m]) and on (node_prefix) (max_over_time(ybp_universe_update_in_progress{node_prefix=""}[31m]) == 0)) > 2
+  ```
+
+- ```expression
+  expr: min by (node_name) (ybp_health_check_c2n_cert_validity_days{universe_uuid=""} < 30)
+  ```
+
+- ```expression
+  expr: last_over_time(ybp_health_check_status{universe_uuid = ""}[1d]) < 1
+  ```
+
+- ```expression
+  expr: ybp_health_check_tserver_version_mismatch{universe_uuid=""} + ybp_health_check_master_version_mismatch{universe_uuid=""} > 0
+  ```
+
+- ```expression
+  expr: count by (node_prefix) (max_over_time(up{export_type="node_export",node_prefix=""}[15m]) < 1) > 0
+  ```
+
+- ```expression
+  expr: last_over_time(ybp_alert_config_writer_status[1d]) < 1
+  ```
+
+- ```expression
+  expr: last_over_time(ybp_alert_query_status[1d]) < 1
+  ```
+
+- ```expression
+  expr: last_over_time(ybp_health_check_notification_status{universe_uuid = ""}[1d]) < 1
+  ```
+
+- ```expression
+  expr: ybp_health_check_cqlsh_connectivity_error{universe_uuid=""} > 0
+  ```
+
+- ```expression
+  expr: sum by (node_prefix) (increase(leader_memory_pressure_rejections{node_prefix=""}[10m])) + sum by (node_prefix) (increase(follower_memory_pressure_rejections{node_prefix=""}[10m])) + sum by (node_prefix) (increase(operation_memory_pressure_rejections{node_prefix=""}[10m])) > 0
+  ```
+
+- ```expression
+  expr: last_over_time(ybp_alert_manager_channel_status{customer_uuid = ""}[1d]) < 1
+  ```
+
+- ```expression
+  expr: min by (node_name) (ybp_health_check_c2n_ca_cert_validity_days{universe_uuid=""} < 30)
+  ```
+
+- ```expression
+  expr: max by (node_prefix) (yb_node_is_master_leader{node_prefix=""}) < 1
+  ```
+
+- ```expression
+  expr: ybp_universe_inactive_cron_nodes{universe_uuid = ""} > 0
+  ```
+
+- ```expression
+  expr: count by (universe_uuid) (ybp_health_check_used_fd_pct{universe_uuid=""} > 70)
+  ```
+
+- ```expression
+  expr: max by (node_prefix) (max_over_time(hybrid_clock_skew{node_prefix=""}[10m])) / 1000 > 500
+  ```
+
+- ```expression
+  expr: last_over_time(ybp_create_backup_status{universe_uuid = ""}[1d]) < 1
```

Comment:
Looks good overall.
The main concern I have with these is that these expressions may change over time (as we'll be making them more precise and exclude some scenarios, where we don't want to raise an alert, for example universe operations)
Also, we'll be adding more alerts, which will make it hard to maintain this list over time.
I think we may want to just implement a feature to open prometheus UI with corresponding expression from alert/alert configuration - which will make it possible to view expression without opening docs + these will never become outdated.

Reply:
But, again, until we have this functionality - it's probably good to have these documented